src/index.html

```diff
@@ -10,13 +10,102 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html">
+        <img
+          src="/src/images/logo.png"
+          alt="logo"
+          class="nav__logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list-items">
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. There should be empty lines between the `<li>` elements.